smdebug/tensorflow/base_hook.py

```diff
@@ -253,13 +253,9 @@ def export_collections(self):
     def has_default_hook_configuration(self):
         # Used in AWS TF to determine if the hook
         # is using the default hook configuration
-        if not self.prepared_collections:
-            self._prepare_collections()
-
-        collections_being_saved = [x.name for x in self._collections_to_save]
-        if set(collections_being_saved) == set(TF_DEFAULT_SAVED_COLLECTIONS):
-            return True
-        return False
+        return super().has_default_hook_configuration(
+            default_saved_collections=TF_DEFAULT_SAVED_COLLECTIONS
+        )
```

Comment:
Irrelevant change. Refactoring so that the default collections are passed in as an argument from KerasHook and XGBoost Hook instead of the implementation being completely duplicated.